tests/test_single.py

```diff
@@ -238,6 +242,37 @@ def test_tx_status(http1_client):
     assert status == "Invalid"
 
 
+def test_public_prefix(governance_client):
```

Comment:
I think it'd be nice to use the `ccf.Ledger` API to check that public keys are visible in clear in the ledger?

Reply:
Yes! That's very nice, great to have the Python API for that :)